webapi/_test_data/ontologies/biblio-onto.ttl

```diff
@@ -9,6 +9,7 @@
 @prefix event: <http://purl.org/NET/c4dm/event.owl#> .
 @prefix knora-base: <http://www.knora.org/ontology/knora-base#> .
 @prefix salsah-gui: <http://www.knora.org/ontology/salsah-gui#> .
+@prefix beol: <http://www.knora.org/ontology/beol#> .
 @base <http://www.knora.org/ontology/biblio-onto> .
```

Comment:
Please check the prefixes for the biblio ontology. I think it should just say <http://www.knora.org/ontology/biblio> (without "onto").